src/main/java/redis/clients/jedis/resps/KeyedListElement.java

```diff
@@ -0,0 +1,45 @@
+package redis.clients.jedis.resps;
+
+import redis.clients.jedis.util.SafeEncoder;
+
+public class KeyedListElement {
+
+  private final String key;
+  private final String element;
+
```

Comment:
I am thinking about it is possible to do with generic?

```
public class KeyedListElement<K, E> {

  private final K key;
  private final E element;
```

Reply:
It is a matter of discussion. I avoided using generics for just two commands.

Reply:
@dengliming why do you think it should be generic? when do you see it used not as String String?